### ISSUE TITLE
Markdown styling needs improvement and can result in an unhandled error

### ISSUE THREAD

**Issue by divin** [reflex-dev/reflex#1835]
**Describe the bug**
While using the `rx.markdown` function to render markdown text I noticed that styling of the `blockquote` tag as well as the `code` tag was not working. I have provided the styling via the `custom_styles` prop. 

In a larger project I encouter an unhandled error when including the `blockquote` tag or the `code` tag in the `custom_styles` prop. This was the error in the Javascript Console: `TypeError: Cannot read properties of undefined (reading 'inTable')`

I couldn't reproduce the error in a smaller project, but the styling is still not working.

**To Reproduce**
This is a simple example which shows the problem of not being able to style the `blockquote` and `code` tag:

```python
import reflex as rx

class State(rx.State):
    pass

text = f"""
    # Welcome to Reflex!
    
    > This is a quote
    
    ```python
    # This is a code block
    import reflex as rx
    ```
"""

def index() -> rx.Component:
    return rx.markdown(
        text,
        custom_styles={
            "blockquote": {
                "line_height": "1.5rem",
                "background": "red",
                "margin_bottom": "1rem",
                "padding": "1rem",
                "max_width": "50rem",
            },
            "code": {
                "line_height": "1.5rem",
                "margin_bottom": "1.0rem",
                "max_width": "960px",
            },
        }
    )

# Add state and page to the app.
app = rx.App()
app.add_page(index)
app.compile()
``````

**Expected behavior**
That the styles are applied to the `blockquote` and `code` tag.

**Screenshots**
This is the styling of larger project:
![Screenshot_2023-09-18_at_20 30 33](https://github.com/reflex-dev/reflex/assets/71128266/f8094b86-98cf-4401-872e-7f3b4afbda5b)
This is the error I got:
![Screenshot_2023-09-18_at_20 31 39](https://github.com/reflex-dev/reflex/assets/71128266/05b874b8-0562-458b-9410-716a9c4b13ab)

**Specifics (please complete the following information):**
 - Python Version: 3.10.12
 - Reflex Version: 0.2.7
 - OS: macOS Ventura
 - Browser (Optional): Safari

**Comment by picklelo** [comment]
@divin thanks for raising this - the error should be solved by #1837 - we weren't pinning some of these markdown dependencies and they had a recent breaking update. Will look into the custom styling issues next.

**Comment by Lendemor** [comment]
Closing this issue as the PR fixing it is merged. Feel free to re-open after next release if the issue persist.
